_partials/nav.html.haml

```diff
@@ -34,6 +34,8 @@
           %ul.dropdown-menu
             %li<
               %a(href="/documentation") Documentation
+            %li<
+              %a(href="/books") Books
```

Comment:
This title doesn't quite confer the fact that it contains books for "old versions". Maybe remove it from the menu, and add it as a button "Books for older versions" in the main documentation page.

Reply:
thanks @tristantarrant I'm pretty happy with this now and have removed the lorem ipsum so everything should be in place - apart from getting things wired in under the hood